tests/cn-runtime/cn-runtime-dir.sh

```diff
@@ -1,97 +1,71 @@
-#! /bin/bash
+#!/usr/bin/env bash
+set -euo pipefail -o noclobber
 
-# tests for the CN executable spec tool attached to cerberus
+function echo_and_err() {
+    printf "$1\n"
+    exit 1
+}
 
-DIRNAME=$1
+[ $# -eq 0 ] || echo_and_err "USAGE: $0"
 
-SUCC=$(find $DIRNAME -maxdepth 1 -name '*.c' | grep -v '\.error\.c' | grep -v 'fixme_error' | grep -v '\.unknown\.c' | grep -v '\-exec\.c' | grep -v 'inconsistent*' | grep -v 'implies*')
-# SUCC=$(find $DIRNAME -maxdepth 1 -name '*tree_rev01.c')
+RUNTIME_PREFIX="$OPAM_SWITCH_PREFIX/lib/cn/runtime"
+[ -d "${RUNTIME_PREFIX}" ] || echo_and_err "Could not find CN's runtime directory (looked at: '${RUNTIME_PREFIX}')"
 
-NUM_GENERATION_FAILED=0
-GENERATION_FAILED=''
+CHECK_SCRIPT="${RUNTIME_PREFIX}/libexec/cn-runtime-single-file.sh"
 
-NUM_COMPILATION_FAILED=0
-COMPILATION_FAILED=''
+[ -f "${CHECK_SCRIPT}" ] || echo_and_err "Could not find single file helper script: ${CHECK_SCRIPT}"
```

Comment:
I think this is a copy paste error.

Reply:
I also tried running the script locally and it seems to be working well

Reply:
No as in, this is the "dir" script but the string says "single file script"

Reply:
Ah, got it. No, this is correct because it's looking for the single file helper script within the Cerberus runtime library regardless of what the top-level script (in this case, the `cn-runtime-dir` script I pushed) is doing. It's `cn-runtime-single-file` which is being searched for `$OPAM_SWITCH_PREFIX/lib/cn/runtime/libexec` and if it's not found, this is the error that gets thrown.